lib/Doctrine/ORM/EntityRepository.php

```diff
@@ -57,6 +58,9 @@ class EntityRepository implements ObjectRepository, Selectable
      */
     protected $_class;
 
+    /** @var Inflector */
```

Comment:
@greg0ire i really have to stop using Vim to do my Doctrine changes :) I don't have inflector type imported already, is it really necessary to make the use statement just for the docblock? seems excessive :)

Reply:
I use Vim too, and it is absolutely terrific! Just make sure you use `phpactor` with it, and use `:PhpactorImportClass` ;) My vimrc is here if you want to take a look: https://github.com/greg0ire/dotvim/blob/master/vimrc

> I don't have inflector type imported already, is it really necessary to make the use statement just for the docblock? seems excessive :)

I don't make the rules, it's our coding standard… anyway, the docblock will become a type in php 7.4, which means we will end up importing the use statement if we didn't already. Let's do it now, it's just less work.